### PR TITLE
Define Sn as a HIT

### LINKED ISSUE #8: Write out the definion of Coxeter relation as a HIT

### PR DESCRIPTION
**Pull request by vikraman**
Fixes #8

### CODE REVIEW

**Comment by inexxt** [comment]
I wouldn't say I undestand the whole code, but - why is it not parametrised by `n`?
(Such as the relation we've defined in here https://github.com/vikraman/2DTypes/commit/b6b6c35c255622294397bb188c14e579a1a88629 ?)

**Comment by inexxt** [comment]
I changed the code to be parametrised by `n` (e.g. in Fin n instead of ℕ) - jus to show what I meant, not as a final statement. Please modify it as you like.

File: Pi+/Coxeter/CoxeterHIT.agda [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    f-swap-β : {n : Fin (S m)} {k : Fin (S m)} → (p : S (k .fst) < (n .fst)) {w : CList} → ap f (swap {n} {k} p {w}) == swap* p {f w}
+    f-swap-β p = apd=cst-in (E.f-swap-β p)
+
+    f-braid-β : {n : Fin m} {w : CList} → ap f (braid {n} {w}) == braid* {n} {f w}
+    f-braid-β = apd=cst-in E.f-braid-β
+
+-- infixr 50 _++_
+
+-- _++_ : ∀ {m} → CList m → CList m → CList m
+-- _++_ {m} = CListRec.f (λ w → w) (λ n f w → n :: f w) (λ= λ _ → cancel) (λ p → λ= λ _ → swap p) (λ= λ _ → braid)
```
Comment (inexxt) [inline Pi+/Coxeter/CoxeterHIT.agda:105]: I don't really understand how to deal with HITs in Agda, so just commented these out.

**Review by inexxt — CHANGES_REQUESTED** [review]

File: Pi+/Coxeter/CoxeterHIT.agda (hunk 1 above)
Comment (inexxt) [reply Pi+/Coxeter/CoxeterHIT.agda]: Fixed.

**Comment by vikraman** [comment]
Wouldn't it be better to just expand out the `Fin`s like in `LL` ?

**Review by inexxt — APPROVED** [review]

**Pull request merged**